apps/src/JSInterpreter.js

```diff
@@ -104,6 +105,9 @@ var JSInterpreter = module.exports = function (options) {
   };
 
   try {
+    // Return value will be stored as this.interpreter inside the supplied
+    // initFunc() (other code in initFunc() depends on this.interpreter, so
+    // we can't wait until the constructor returns)
```

Comment:
Which code in `initFunc()` depends on `this.interpreter` being set?  On the surface, it looks like it all uses the created interpreter that's passed as an argument to `initFunc()` - which is probably what it should be doing, right?